### DISCUSSION TITLE
Errors in phoebe's calculation

### DISCUSSION THREAD

**Discussion by astronomical-beginner** [phoebe-project/phoebe2#962] [Questions]
Hello, developer
I encountered an error when I used PHOEBE to calculate. I reinstalled PHOEBE several times and still can't solve it. I am very worried about this, and I hope you or others can help me. Thank you very much!

![error](https://github.com/user-attachments/assets/a104228c-61e2-4532-a24d-58306191f709)

**Comment by Raindogjones** [comment]
This looks like it might be a problem with one of the passband files, can you post the previous lines of code to see what passband you are using? Then we might be able to track down the problem.

**Reply by astronomical-beginner** [reply]
I'm so sorry for not describing it clearly.The passband I used is 'TESS: T'.

**Reply by Raindogjones** [reply]
Can you try running the following code and post the output.
```
import phoebe
phoebe.__version__
pb=phoebe.get_passband("TESS:T")
pb.content
```

**Reply by astronomical-beginner** [reply]
I tried this code, but the problem still didn't solve.

![error1](https://github.com/user-attachments/assets/7145e144-90fc-4cc1-8d13-09267a1b82b5)
![error2](https://github.com/user-attachments/assets/6b73624d-7eda-4f5c-b1f2-ce0124b7c52c)
![error3](https://github.com/user-attachments/assets/1af30919-7ae6-4382-8f81-b2b9daf4279d)

**Reply by Raindogjones** [reply]
Sorry, I should have specified to run the first two lines separately so as to get the version of phoebe you are running.
`import phoebe`
`phoebe.__version__`

**Reply by astronomical-beginner** [reply]
Thank you for your explanation.
I tried this way to run, but it still doesn't seem to work.
![error4](https://github.com/user-attachments/assets/45c0a182-f1b1-4886-9829-945cbdb7ad32)
![error5](https://github.com/user-attachments/assets/465154ec-4297-4466-b64f-bc2fe80b4f16)

**Reply by Raindogjones** [reply]
The passband looks complete enough based on the output of `pb.content`...

**Reply by Raindogjones** [reply]
Can you try a much more simplified run to see if the error changes?
`b=phoebe.default_binary()`
`b.add_dataset('lc',times=0,passband='TESS:T')`
`b.run_compute()`

**Reply by astronomical-beginner** [reply]
Thanks.
I think the problem is ‘b.run_compute()’ ,but I don't know the solution.

![error6](https://github.com/user-attachments/assets/4129d907-68c4-4ee3-9135-7b660f99b89b)

![error7](https://github.com/user-attachments/assets/b0a08208-8e84-4791-bbe2-ebbcde8684f9)

**Reply by Raindogjones** [reply]
And if you try a different passband? i.e. Johnson:V

**Reply by astronomical-beginner** [reply]
Thank you, but it still doesn't seem to work.
![error8](https://github.com/user-attachments/assets/f39548f4-8e3f-47fe-bafe-90dedb60d5d2)

**Reply by Raindogjones** [reply]
The problem seems to be with the passband files themselves. Maybe you can try to force a redownload with:
`phoebe.update_passband("Johnson:V")`
Otherwise you could try deleting the passband files and reinstalling them.  They should be stored in one (or more) of the directories listed by:
`phoebe.list_passband_directories()`

**Reply by astronomical-beginner** [reply]
Sorry, the update of the passband takes too long, and even now the update has not been completed. I'll get back to you as soon as it's finished.

**Reply by Raindogjones** [reply]
Sounds like the problem might well be due to incomplete passband files then - hopefully allowing the download to complete will fix things.

**Reply by astronomical-beginner** [reply]
I think so, too, but when I tried to update two kinds of passbands (TESS:T, Johnson:V) to calculate, the results were still unsatisfactory. 
The following figure is the calculation result of passband Tess: T. Due to I calculated the Johnson:V passband first, it is not reflected in the figure, but the results of both are consistent.
![error9](https://github.com/user-attachments/assets/2d5d3240-d027-4b03-a6c3-980f0a1d35cb)

**Reply by astronomical-beginner** [reply]
Later, I tried several passbands, but the results remained unchanged. I think I might have to reinstall the windows system. Thank you for your patient reply!

**Reply by kecnry** [reply]
Sorry for the troubles you're running into - we haven't seen this case before, but I don't think there's any need to reinstall windows!  There is a chance this might be a windows-specific bug, though (if anyone with a windows machine can reproduce, please let us know!).

In the meantime, there's a chance this is also isolated to interpolated limb-darkening.  Can you try setting `b.set_value_all('ld_mode*', 'manual')` before the call to `run_compute` and let us know if that bypasses this error or not?

**Reply by astronomical-beginner** [reply]
Thank you for your reply, and it worked.
![error10](https://github.com/user-attachments/assets/d110da1c-3f9d-467d-a3eb-e5287602c3ad)

**Reply by Raindogjones** [reply]
This makes it look suspiciously like a windows-specific bug to me then - it seems unlikely that all passbands would download complete enough to run without limb-darkening but not enough to run with.

**Reply by kecnry** [reply]
Thanks for confirming, I created #966 to track the bug.  In the meantime, you can either provide manual limb-darkening coefficients (you probably won't want to just leave them at their defaults), or you can try using `ld_mode='lookup'`.  See the [limb-darkening tutorial](https://phoebe-project.org/docs/latest/tutorials/limb_darkening) for more details on these parameters.

**Reply by astronomical-beginner** [reply]
Thank you for your reply! I have solved this problem by reinstalling the windows system at present, and I may not be able to continue to implement your method to solve this bug. I am sorry for this.

**Reply by MichaelAbdul-Masih** [reply]
![issue962_screenshot](https://github.com/user-attachments/assets/102201c1-a616-4842-a5de-50139a56cba5)

I tried to reproduce the bug in my local windows install of PHOEBE and it ran without issue so this looks like it may have been an isolated issue with your installation.  I'm glad that its working now for you though!
